### PR TITLE
Address APIGW limitation inside Java Wrapper Lambda Layer

### PR DESCRIPTION
**Pull request by Kausik-A**
**Description**

This PR adds a Note inside [lambda-java](https://aws-otel.github.io/docs/getting-started/lambda/lambda-java) to indicate only the 1 kind of APIGW event object is supported.

**Issue**

https://github.com/open-telemetry/opentelemetry-lambda/issues/270

### CODE REVIEW

File: src/docs/getting-started/lambda/lambda-java.mdx [hunk 1]
```diff
@@ -26,6 +26,8 @@ With the ADOT Lambda Layer for Java, the wrapper has built-in support for instru
 The Lambda layer supports Java 8 and 11 Lambda runtimes. For more information about supported Java versions,
 see the [OpenTelemetry Java documentation](https://github.com/open-telemetry/opentelemetry-java).
 
+Note: Inside the Java-Wrapper layer only the 1 kind of APIGW (APIGatewayProxyRequestEvent) event object is currently supported.
```
Comment (vasireddy99) [inline src/docs/getting-started/lambda/lambda-java.mdx:29]: Can we move this note under this section : https://github.com/aws-otel/aws-otel.github.io/blob/main/src/docs/getting-started/lambda/lambda-java.mdx#enable-auto-instrumentation-for-your-lambda-function

**Comment by vasireddy99** [comment]
Branch needs to be updated 🙂.

**Comment by vasireddy99** [comment]
Looks like it needs to be done again😿

File: src/docs/getting-started/lambda/lambda-java.mdx [hunk 2]
```diff
@@ -67,6 +67,8 @@ Tips:
  Make sure your Lambda role has the required AWS X-Ray permissions.
   See more on AWS X-Ray permissions for AWS Lambda, see the [AWS Lambda documentation](https://docs.aws.amazon.com/lambda/latest/dg/services-xray.html#services-xray-permissions).
 
+Note: Inside the Java-Wrapper layer only the 1 kind of APIGW (APIGatewayProxyRequestEvent) event object is currently supported.
```
Comment (Aneurysm9) [inline src/docs/getting-started/lambda/lambda-java.mdx:70]: Can we provide any more detail here?  What does this mean for the user?  Is there anything they can do if they are impacted by it?  What about non-APIG events?

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx:70]: We can include the [open issue](https://github.com/open-telemetry/opentelemetry-lambda/issues/270) which has more context and a link to upstream implementation of this specific  `tracingRequestApiGatewayWrapper`. For non-APIG events, it should error out in a similar pattern as mentioned in the issue. Currently there is no workarounds to make it work, other than fixing it upstream.

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx:70]: Lmk if this work, I will update it the comment above

**Comment by Kausik-A** [comment]
Rebased it

**Comment by Kausik-A** [comment]
@Aneurysm9 should I change the wording in accordance to the discussion in the above thread?

File: src/docs/getting-started/lambda/lambda-java.mdx (hunk 2 above)
Comment (bryan-aguilar) [reply src/docs/getting-started/lambda/lambda-java.mdx:70]: @Kausik-A please add a link to the referenced issue.

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx:65]: I have already added it under description

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx:65]: do you mean to add it onto the page?

Comment (bryan-aguilar) [reply src/docs/getting-started/lambda/lambda-java.mdx:65]: Yes, adding a link to an open issue should give readers more context and a central place to look for updates.

Comment (Kausik-A) [reply src/docs/getting-started/lambda/lambda-java.mdx]: Added

**Review by vasireddy99 — APPROVED** [review]
LGTM

**Pull request merged**